src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,43 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  if (typeof(text) !== 'string') {
+    throw "Please enter a valid string";
```

Comment:
Prefer to throw an `Error` instead rather than primitives to preserve the stack trace. In this scenario, `TypeError` is preferred.

Reply:
Noted sir! Thanks!!